src/main/java/com/whitesource/cure/Encoder.java

```diff
@@ -45,12 +45,12 @@ public static String forOsCommand(@NonNull final String param, char[] charsToIgn
    * @param contents arrays {@link Object} contains all the contents.
    * @return encoded log content.
    */
-  public static String[] multiLogContentEncoder(@NonNull final Object[] contents) {
+  public static String[] forMultiLogContent(@NonNull final Object[] contents) {
```

Comment:
This is an overloading so A. change name to also be forLogContent and B. add String[] as an accepted value for sanitation in logInjection fixer

Reply:
there already is a forLogContent method which receives an Object instead of Object[], the IDE says the array method would be linked to the single one upon renaming